project/plugins.sbt

```diff
@@ -4,5 +4,5 @@ logLevel := Level.Warn
 updateOptions := updateOptions.value.withGigahorse(false)
 
 // Publishing
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.7")
\ No newline at end of file
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.2")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
```

Comment:
Neither of these should actually be needed in our case. We don't release this as a signed Maven Repository artifact, nor do we follow that automated release process.